GitCommands/Logging/CommandLog.cs

```diff
@@ -64,23 +67,53 @@ internal CommandLogEntry(string fileName, string arguments, DateTime startedAt,
             IsOnMainThread = isOnMainThread;
         }
 
-        public override string ToString()
+        public string ColumnLine
         {
-            var duration = Duration == null
-                ? "running"
-                : $"{((TimeSpan)Duration).TotalMilliseconds:0}ms";
+            get
+            {
+                var duration = Duration == null
+                    ? "running"
+                    : $"{((TimeSpan)Duration).TotalMilliseconds:0}ms";
 
-            var fileName = FileName;
+                var fileName = FileName;
 
-            if (fileName.EndsWith("git.exe"))
-            {
-                fileName = "git";
-            }
+                if (fileName.EndsWith("git.exe"))
+                {
+                    fileName = "git";
+                }
 
-            var pid = ProcessId == null ? "     " : $"{ProcessId,5}";
-            var exit = ExitCode == null ? "  " : $"{ExitCode,2}";
+                var pid = ProcessId == null ? "     " : $"{ProcessId,5}";
+                var exit = ExitCode == null ? "  " : $"{ExitCode,2}";
 
-            return $"{StartedAt:HH:mm:ss.fff} {duration,7} {pid} {(IsOnMainThread ? "UI" : "  ")} {exit} {fileName} {Arguments}";
+                return $"{StartedAt:HH:mm:ss.fff} {duration,7} {pid} {(IsOnMainThread ? "UI" : "  ")} {exit} {fileName} {Arguments}";
+            }
+        }
+
+        public string Detail
+        {
+            get
+            {
+                var s = new StringBuilder();
+
+                s.Append("File name:  ").AppendLine(FileName);
+                s.Append("Arguments:  ").AppendLine(Arguments);
+                s.Append("Process ID: ").Append(ProcessId == null ? "unknown" : ProcessId.ToString()).AppendLine();
+                s.Append("Started at: ").Append(StartedAt.ToString("O")).AppendLine();
+                s.Append("UI Thread?: ").Append(IsOnMainThread).AppendLine();
+                s.Append("Duration:   ").Append(Duration == null ? "still running" : $"{Duration.Value.TotalMilliseconds:0.###} ms").AppendLine();
+                s.Append("Exit code:  ").Append(ExitCode == null ? "unknown" : ExitCode.ToString()).AppendLine();
+
+                if (CallStack != null)
+                {
+                    s.AppendLine("Call stack: ").Append(CallStack);
+                }
+                else
+                {
+                    s.Append("Call stack: not captured");
```

Comment:
Should these strings be localisable?

Reply:
Debug functionality like this does not have to be translated. It may be nicer though.

Reply:
@RussKie, sorry I missed your question here about localisation. I can file a follow up PR if you like.